### PR TITLE
Add beta tag to audit token docs

### PR DESCRIPTION
**Pull request by tstormk**
### What
Adds a `Beta` tag to documentation concerning the new audit trails token feature.

### Why
The audit trails token is currently beta, and while this is made clear in the Atlas UI, it is not currently indicated in the documentation for the feature.

----------

### Merge Checklist
_If items do not apply to your changes, add (N/A) and mark them as complete._

#### Pull Request
- [x] One or more labels describe the type of change (e.g. clarification) and associated product (e.g. HCP Terraform ).

#### Content
- [x] API documentation and the API Changelog have been updated. 
- [ ] The Vercel website preview successfully deployed.

#### Reviews
- [x] I or someone else reviewed the content for technical accuracy.
- [x] I or someone else reviewed the content for typos, punctuation, spelling, and grammar.

### CODE REVIEW

**Review by carolinaborim — APPROVED** [review]

File: website/docs/cloud-docs/api-docs/audit-trails-tokens.mdx [hunk 1]
```diff
@@ -36,10 +36,12 @@ description: >-
 
 [JSON API error object]: https://jsonapi.org/format/#error-objects
```
Comment (rkoron007) [inline website/docs/cloud-docs/api-docs/audit-trails-tokens.mdx:38]: Howdy! Can you also add a Beta tag to the [Navigation file](https://github.com/hashicorp/terraform-docs-common/blob/c875e000e7d36da4f51896f347819487f867c1fe/website/data/cloud-docs-nav-data.json#L50), by changing the title to: `Audit Trails Tokens <sup>Beta</sup>`?

File: website/docs/cloud-docs/api-docs/audit-trails-tokens.mdx [hunk 2]
```diff
@@ -36,10 +36,12 @@ description: >-
...
 
-# Audit Trail Tokens API
+# Audit Trail Tokens API (Beta)
 
 Audit trail tokens are used to authenticate integrations pulling audit trail data, for example, using the [HCP Terraform for Splunk](/terraform/cloud-docs/integrations/splunk) app.
 
+This feature is currently in beta.
```
Comment (rkoron007) [inline website/docs/cloud-docs/api-docs/audit-trails-tokens.mdx:43]:
```suggestion
Audit trail tokens are currently in beta.
```

**Review by rkoron007 — APPROVED** [review]
Looks good! ✨ Added one more place to add a `beta` notice! 

Do you mind if I ask what happened with this one, was this in public beta when we originally merged these docs and we forgot to mark it as so?

**Comment by tstormk** [comment]
@rkoron007 I've updated the PR as you suggested!

**Comment by rkoron007** [comment]
No problemo! This looks good to go for me, you can feel free to merge whenever you'd like ✨

**Pull request merged**